src/drivers/mct/cime_config/config_component.xml

```diff
@@ -3003,6 +3003,23 @@
    <desc>TRUE=>turn on CPP variable COMPARE_TO_NUOPC</desc>
  </entry>
 
+ <entry id="NGPUS_PER_NODE">
```

Comment:
Should this be implemented on a per component basis?  See the NTASKS and NTHRDS variables for example.

Reply:
Thanks @jedwards4b . I have changed this in my work branch and it will be reflected in another PR.